tests/e2e/testfiles/storage_csm_powerscale_resiliency.yaml

```diff
@@ -293,20 +293,3 @@ spec:
             - "--driver-config-params=/csi-isilon-config-params/driver-config-params.yaml"
             - "--driverPodLabelValue=dell-storage"
             - "--ignoreVolumelessPods=false"
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: isilon-config-params
-  namespace: isilon
-data:
-  driver-config-params.yaml: |
-    CSI_LOG_LEVEL: "debug"
-    CSI_LOG_FORMAT: "TEXT"
-    PODMON_CONTROLLER_LOG_LEVEL: "debug"
-    PODMON_CONTROLLER_LOG_FORMAT: "TEXT"
-    PODMON_NODE_LOG_LEVEL: "debug"
-    PODMON_NODE_LOG_FORMAT: "TEXT"
-spec:
-  driver:
-    configVersion: v2.11.0
```

Comment:
Why are we removing this configmap? What did it do that was wrong? What does removing it unblock?